src/main.rs

```diff
@@ -18,6 +18,11 @@ struct Args {
     #[clap(long)]
     interp: bool,
 
+    /// Translate the input program to a PEG and run it.
+    /// Throws an error if used with the interp flag.
+    #[clap(long)]
+    peg_interp: bool,
```

Comment:
We don't need this command line flag because we already have the `run-mode` and `interp` flags, which should already just work!

Reply:
`interp` means "run the optimized Bril", `peg-interp` means "run the unoptimized PEG". I think that it's worth separating the flags because of the optimized/unoptimized distinction, not the Bril/PEG distinction.

Reply:
Actually, I usually think of `interp` as "run the result". The `RunType` determines if it is optimized or not optimized